src/csi-driver-nfs/chart/values.yaml

```diff
@@ -4,3 +4,12 @@ sidecars: {}
 csinfs: {}
 # baseRepo: "anotherrepo"
 # dockerRegistrySecret: "anothersecret"
+
+nodeSelector: {}
+tolerations: []
+affinity: {}
+
+plugin:
```

Comment:
We don't really need two sets of selectors. Please remove plugin block alongwith nodeSelector and affinity